cms/djangoapps/contentstore/helpers.py

```diff
@@ -323,6 +323,58 @@ def import_staged_content_from_user_clipboard(parent_key: UsageKey, request) ->
     return new_xblock, notices
 
 
+def fetch_and_set_upstream_link(
```

Comment:
This function seems pretty local to `_import_xml_node_to_parent`.. should it be an internal function to discourage use outside of this module?
```suggestion
def _fetch_and_set_upstream_link(
```

Reply:
Makes sense. Updated.